### ISSUE TITLE
pc miner

### ISSUE THREAD

**Issue by Naveen8239** [duino-coin/duino-coin#801]
my miners on but not show in wallet sloved that
![Screenshot_2021-08-13-09-48-42-56_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/88766353/129304403-11e110ff-0d17-4989-abfc-9f9113fc2cb7.jpg)

**Comment by KingUndeadCodes** [comment]
1. Please do not open a new issue. Use the one you opened before.
2. You should ask in the discord server.

**Comment by revoxhere** [comment]
Should be fixed
